src/actions/profile-view.js

```diff
@@ -1169,42 +1164,6 @@ export function changeProfileName(profileName: string): Action {
   };
 }
 
-export function changeShowTabOnly(
```

Comment:
Removing this action now. We are gonna use `changeViewAndRecomputeProfileData` instead.